src/main/java/org/bytedeco/javacpp/tools/Generator.java

```diff
@@ -525,14 +531,14 @@ boolean classes(boolean handleExceptions, boolean defineAdapters, boolean conver
         out.println("}");
         out.println();
         out.println("#if !defined(NO_JNI_DETACH_THREAD) && defined(_WIN32)");
-        out.println("   static __declspec(thread) struct JavaCPP_thread_local {");
+        out.println("   static struct JavaCPP_thread_local {");
         out.println("       JNIEnv* env = NULL;");
         out.println("       ~JavaCPP_thread_local() {");
         out.println("           if (env && JavaCPP_vm) {");
         out.println("               JavaCPP_vm->DetachCurrentThread();");
         out.println("           }");
         out.println("       }");
-        out.println("   } JavaCPP_thread_local; ");
+        out.println("   } THREADLOCAL JavaCPP_thread_local; ");
```

Comment:
It looks like it's more standard to use that keyword before the type, for example:
https://en.cppreference.com/w/cpp/language/storage_duration
It's like a modifier of the `static` keyword.